compiler-rt/lib/builtins/cpu_model/x86.c

```diff
@@ -139,20 +139,77 @@ enum ProcessorFeatures {
   FEATURE_AVX512BITALG,
   FEATURE_AVX512BF16,
   FEATURE_AVX512VP2INTERSECT,
+  FEATURE_3DNOW,
+  FEATURE_ADX = 40,
+  FEATURE_CLDEMOTE = 42,
```

Comment:
Why do we skip some of features?

Reply:
The missing features are ones I can't recognize in https://github.com/gcc-mirror/gcc/blob/02948ced062b730e2859f718dd46bfcaa3c16c6b/gcc/common/config/i386/i386-cpuinfo.h#L154, e.g. FEATURE_ABM, I can't find relate one in LLVM.

Reply:
agree, done in 10c2a7c

Reply:
ABM is a AMD specific feature https://en.wikipedia.org/wiki/X86_Bit_manipulation_instruction_set
Maybe LLVM doesn't support it. But we should be able to map it to its corresponding [CPUID bit](https://github.com/gcc-mirror/gcc/blob/02948ced062b730e2859f718dd46bfcaa3c16c6b/gcc/config/i386/cpuid.h#L56). Maybe not in this patch.
I'd expect we can list the missing parts with comments like
```
FEATURE_3DNOW,
FEATURE_ADX = 40,
// FEATURE_ABM,
FEATURE_CLDEMOTE = 42,
```

Reply:
Good idea. Addressed in 098a413